testing/unittests/esdltests.cpp

```diff
@@ -1844,6 +1870,220 @@ constexpr const char * result = R"!!(<soap:Envelope xmlns:soap="http://schemas.x
             CPPUNIT_ASSERT(false);
         }
     }
+    void testMysql()
+    {
+        static constexpr const char * script1 = R"!!(<es:CustomRequestTransform xmlns:es="urn:hpcc:esdl:script" target="Person">
```

Comment:
Is this used?

Reply:
Only for development.  I'll remove these.